.github/pull_request_template.md

```diff
@@ -23,6 +23,7 @@
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I've added this Pull Request to the AZ Quickstart project in the right column.
```

Comment:
We should link to the project from here

Reply:
We should also link to the CONTRIBUTING.md document below.

Reply:
[AZ Quickstart project](https://github.com/orgs/az-digital/projects/1)
 [CONTRIBUTING](https://github.com/az-digital/az_quickstart/blob/master/CONTRIBUTING.md)
``` [AZ Quickstart project](https://github.com/orgs/az-digital/projects/1)```
 ```[CONTRIBUTING](https://github.com/az-digital/az_quickstart/blob/master/CONTRIBUTING.md)```

Reply:
This will be reconsidered to also account for ZenHub.